doc/nyan.md

```diff
@@ -347,7 +347,7 @@ the calculation is done like this:
     - `text`:     `"lol"`          - (duh.)
     - `int`:      `1337`           - (some number)
     - `float`:    `42.235`, `inf`  - (some floating point number)
-    - `bool`:     `True`, `False`  - (some boolean value)
+    - `bool`:     `true`, `false`  - (some boolean value)
```

Comment:
[minor] missed the following usages in the examples below, find/replace all.
Lines: 1086, 1097, and 1146.

Reply:
fix'd